framework/auth/__init__.py

```diff
@@ -117,20 +117,17 @@ def verify_two_factor(user_id, two_factor_code):
     return response
 
 
-def login(username, password):
+def login(email, password):
```

Comment:
Just curious: why was this changed? The field name is called `username` so it sees intuitive for the parameter name to be the same.